ci/build_test_OnCommitWin.groovy

```diff
@@ -0,0 +1,40 @@
+pipeline {
+    options {
+        timeout(time: 2, unit: 'HOURS')
+    }
+    agent {
+        label 'win_ovms'
+    }
+    stages {
+        stage ("Clean") {
+            steps {
+                script{
+                    def output1 = bat(returnStdout: true, script: 'clean_windows.bat')
+                }
+            }
+        }
+        // Build windows
+        stage("Build windows") {
```

Comment:
Why do we have similar stage in non-windows groovy?

Reply:
We trigger windows groovy job on jenkins from our oncommit groovy.